packaging/docker/release/Dockerfile

```diff
@@ -20,27 +20,28 @@
 FROM ubuntu:18.04
 
 RUN apt-get update && \
-	apt-get install -y curl>=7.58.0-2ubuntu3.6 \
-		dnsutils>=1:9.11.3+dfsg-1ubuntu1.7 \
-		lsof>=4.89+dfsg-0.1 \
-		tcptraceroute>=1.5beta7+debian-4build1 \
-		telnet>=0.17-41 \
-		netcat>=1.10-41.1 \
-		strace>=4.21-1ubuntu1 \
-		tcpdump>=4.9.3-0ubuntu0.18.04.1 \
-		less>=487-0.1 \
-		vim>=2:8.0.1453-1ubuntu1.4 \
-		net-tools>=1.60+git20161116.90da8a0-1ubuntu1 \
-		jq>=1.5+dfsg-2 && \
+	apt-get install -y curl \
+		dnsutils \
+		lsof \
+		tcptraceroute \
+		telnet \
+		netcat \
+		strace \
+		tcpdump \
+		less \
+		vim \
+		net-tools \
+		jq && \
 	rm -rf /var/lib/apt/lists/*
 
-COPY misc/tini-amd64.sha256sum /tmp/
```

Comment:
I think it was intended not to download the sha256sum from somewhere else but I don't recall why. The initial setup already did that: https://github.com/apple/foundationdb/pull/4363/files#diff-028a7f6bae25b75ad5b6f936e31c3d30bf50ce15af5cc2be41cf23af5cdd7805R91

Reply:
Maybe @ammolitor can shed some light.

Reply:
I tried building the images locally but couldn't since the checksum file was missing. Will it make more sense to include the sha256sum file in the repo?